clients/python-apiserver-client/python_apiserver_client/kuberay_apis.py

```diff
@@ -0,0 +1,261 @@
+import requests
+import time
+
+from .params import *
+
+
+_headers = {"Content-Type": "application/json", "accept": "application/json"}
+
+
+class KubeRayAPIs:
+    """
+        This class implements KubeRay APIs based on the API server.
+        To create a classe, the following parameters are required:
+            base - the URL of the API server (default is set to the standalone API server)
+            wait interval - the amount of sec to wait between checking for cluster ready
+    """
+    def __init__(self, base: str = "http://localhost:8888",
+                 wait_interval: int = 2) -> None:
+        self.base = base
+        self.wait_interval = wait_interval
+        self.api_base = "/apis/v1/"
+
+    """
+        List compute templates across all namespaces of the k8 cluster 
+        Returns:
+            http return code
+            message - only returned if http return code is not equal to 200
+            list of compute templates
```

Comment:
Is "template" a well-understood K8s concept here? If so, ignore this comment ;) Otherwise, can you explain what they are for somewhere?